en/lessons/ecto/querying.md

```diff
@@ -195,22 +196,24 @@ First, we'll write a query expression using the `first/2` function:
 
 ```elixir
 iex> first(Movie)
-%Ecto.Query<from m in Friends.Movie, order_by: [desc: m.id], limit: 1>
+#Ecto.Query<from m0 in Friends.Movie, order_by: [asc: m0.id], limit: 1>
 ```
 
-Then we pass our query to the `Repo.one/2` function to get our result:
+However, we will use the pipe with the function `Repo.one/2` to obtain our result:
 
 ```elixir
-iex> Movie |> first() |> Repo.one()
+iex> Movie \
+...>  |> first \
+...>  |> Repo.one
```

Comment:
```suggestion
```

Reply:
This sample was just to show another way to pipe and for me I prefer. 
Sorry about to put something different on the original tutorial.